Blockzilla.xcodeproj/project.pbxproj

```diff
@@ -96,6 +96,7 @@
 		D3E54FD51DEFB25B003E1AFF /* SearchPlugins in Resources */ = {isa = PBXBuildFile; fileRef = D3E54FD41DEFB25B003E1AFF /* SearchPlugins */; };
 		D3E54FDE1DF0E0D7003E1AFF /* UIImageExtensions.swift in Sources */ = {isa = PBXBuildFile; fileRef = D3E54FDD1DF0E0D6003E1AFF /* UIImageExtensions.swift */; };
 		D3E54FE01DF0E221003E1AFF /* SearchSettingsViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = D3E54FDF1DF0E221003E1AFF /* SearchSettingsViewController.swift */; };
+		D426B2281EE0DE1B00C256F6 /* AdjustSdk.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = D426B2271EE0DE1B00C256F6 /* AdjustSdk.framework */; };
```

Comment:
We cannot statically link to Adjust because of Klar. There is already a build step where we run a shell script to copy in the frameworks for Focus *only*: https://github.com/mozilla-mobile/focus/blob/master/Blockzilla.xcodeproj/project.pbxproj#L990-L1004